components/clp-py-utils/clp_py_utils/sql_adapter.py

```
@@ -58,8 +59,11 @@ def alive(self):
 
 
 class SqlAdapter:
-    def __init__(self, database_config: Database):
+    def __init__(
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)